core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -2477,7 +2477,7 @@ public boolean isSCIMEnabled() {
     /**
      * {@inheritDoc}                  doAddInternalRole
      */
-    public final String[] getHybridRoles() throws UserStoreException {
+    public String[] getHybridRoles() throws UserStoreException {
```

Comment:
It is meaningful to have these method final in abstract class as we won't be modifying this method in other implementations on getting the internal roles